logstash-core/lib/logstash/webserver.rb

```diff
@@ -5,8 +5,7 @@
 require 'puma/configuration'
 require 'puma/commonlogger'
 
-module LogStash
-  class WebServer
+module LogStash class WebServer
```

Comment:
why this change? it looks to me a bit cosmetic, before it was actually more clear to understand, right?